en/docs/learn/integration-tutorials/sending-a-simple-message-to-a-datasource.md

```diff
@@ -1,43 +1,50 @@
-# How to Expose a Datasource as a Service
+# Build your first Data Service
 
 ## What you'll build
 
-A **data service** provides a web service interface to access data that is stored in various data sources. The following sections describe how you can use Micro Integrator Extension for Visual Studio Code (MI for VS Code) to work with data services' artifacts. 
+In this tutorial, you will create a [Data Service]({{base_path}}/reference/synapse-properties/data-services/) that exposes an RDBMS data source as a RESTful API.
 
-!!! Tip
-    Note that this feature is currently supported in Micro Integrator Extension for Visual Studio Code (MI for VS Code) for relational data sources and CSV files.
+!!! Tip "What is a Data Service?"
+    A Data Service in WSO2 Micro Integrator allows you to expose data from relational databases, CSV files, or other data sources as RESTful or SOAP APIs. It simplifies integration by handling database queries, input/output mappings, and data transformation without writing custom code. To explore Data Services in detail, see the [Data Services]({{base_path}}/reference/synapse-properties/data-services/) documentation.
+
+When a client sends a request to the Micro Integrator, the employee number will be extracted from the request, used in a SQL query to retrieve employee information, and the result will be returned in JSON format.
 
 ## Let's get started!
 
 ### Step 1: Set up the workspace
 
 {!includes/setting-up-workspace-for-mi-for-vscode.md!}
 
-- **MySQL server:** Follow the steps below to set up the MySQL server.
-    1. Install the MySQL server.
-    2. Connect to the MySQL server using the MySQL client. 
-    2. Download the JDBC driver for MySQL from [here](http://dev.mysql.com/downloads/connector/j/). Ensure that you download the JDBC driver version that matches your MySQL server version. You will need this when you configure the MySQL server with the Micro Integrator.
-        
-    3. Create a database named `Employees`.
+#### Set up MySQL server
+
+Follow the steps below to set up the MySQL server.
 
-        ```bash
-        CREATE DATABASE Employees;
-        ```
+1. Install the <a target="_blank" href="https://www.mysql.com/downloads/">MySQL</a> server.
 
-    4. Create a user and grant the user access to the Database.
+2. Connect to the MySQL server using a MySQL client to create and populate the necessary database for this tutorial. Follow the steps below to create the `Employees` database, a new user named `wso2mi`, and the `Employee` table.
```

Comment:
Shall we change the database name here? As both the table and db name are same, and it is not the usual case.

Reply:
Fixed with dfc9462